attributes/default.rb

```diff
@@ -163,7 +163,7 @@
                                                 httpd boost-devel redhat-lsb mlocate mpich-devel openmpi-devel R atlas-devel
                                                 blas-devel fftw-devel libffi-devel openssl-devel dkms mysql-devel libedit-devel
                                                 libical-devel postgresql-devel postgresql-server sendmail mdadm python python-pip
-                                                libgcrypt-devel libevent-devel]
```

Comment:
I would keep this

Reply:
Sorry, i merged before I saw this.

Why keep it?